packages/components/checkbox/src/checkbox-group.ts

```diff
@@ -37,7 +37,7 @@ export class CheckboxGroup<T = unknown> extends FormControlMixin(LitElement) {
 
   /** Observe changes to the checkboxes. */
   #observer = new MutationObserver(() => {
```

Comment:
Can you please update the example on the website? Right now it's not rendering
<img width="965" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/1ca12f01-551b-42cd-8eff-b68b4bf810ed">

<img width="618" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/f069bce7-c294-4b21-a866-11c506d4b4d6">

Reply:
Fixed